<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sacvision 0.1.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sacvision.qrc">
    <normaloff>:/images/mainlogo.png</normaloff>:/images/mainlogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sacvision.qrc">:/images/logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SacHistoWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionLicense"/>
    <addaction name="actionLibraries"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionAcquire"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionEdge_Detection"/>
    <addaction name="actionSmooth"/>
    <addaction name="actionFFT"/>
    <addaction name="actionComplex"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperations"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionClose"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionAcquire"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionSmooth"/>
   <addaction name="actionEdge_Detection"/>
   <addaction name="actionComplex"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Work with the default image</string>
   </property>
   <property name="whatsThis">
    <string>Work with the default image</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open an image to work with</string>
   </property>
   <property name="whatsThis">
    <string>Open an image to work with</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the current image</string>
   </property>
   <property name="whatsThis">
    <string>Save the current image</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save As..</string>
   </property>
   <property name="statusTip">
    <string>Set a name and save the image</string>
   </property>
   <property name="whatsThis">
    <string>Set a name and save the image</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="folder"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close the work with the image</string>
   </property>
   <property name="whatsThis">
    <string>Close the work with the image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Go back to the OS</string>
   </property>
   <property name="whatsThis">
    <string>Go back to the OS</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="applications-utilities"/>
   </property>
   <property name="text">
    <string>AboutQt</string>
   </property>
   <property name="toolTip">
    <string>Show the Qt License</string>
   </property>
   <property name="statusTip">
    <string>Show the Qt License</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Sacvision</string>
   </property>
   <property name="statusTip">
    <string>About Sacvision</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset theme="applications-development"/>
   </property>
   <property name="text">
    <string>License</string>
   </property>
   <property name="toolTip">
    <string>Show the license to use Sacvision</string>
   </property>
   <property name="statusTip">
    <string>Show the license th use Sacvision</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy the input image to the output one</string>
   </property>
   <property name="statusTip">
    <string>Copy the input image to the output one</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset theme="media-playback-start"/>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the selected processing</string>
   </property>
   <property name="statusTip">
    <string>Start the selected processing</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="media-playback-stop"/>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the selected processing</string>
   </property>
   <property name="statusTip">
    <string>Stop the selected processing</string>
   </property>
  </action>
  <action name="actionLibraries">
   <property name="icon">
    <iconset theme="drive-harddisk"/>
   </property>
   <property name="text">
    <string>Libraries</string>
   </property>
  </action>
  <action name="actionEdge_Detection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="network-wireless"/>
   </property>
   <property name="text">
    <string>Edge Detection</string>
   </property>
   <property name="toolTip">
    <string>Apply and edge detection filter</string>
   </property>
   <property name="statusTip">
    <string>Apply and edge detection filter</string>
   </property>
  </action>
  <action name="actionSmooth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="image-x-generic"/>
   </property>
   <property name="text">
    <string>Smooth</string>
   </property>
   <property name="iconText">
    <string>Smooth the input image</string>
   </property>
   <property name="toolTip">
    <string>Smooth the imput image</string>
   </property>
  </action>
  <action name="actionComplex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-restore"/>
   </property>
   <property name="text">
    <string>Complex</string>
   </property>
   <property name="toolTip">
    <string>Apply a complex filter</string>
   </property>
   <property name="statusTip">
    <string>Apply a complex filter</string>
   </property>
  </action>
  <action name="actionFFT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="utilities-system-monitor"/>
   </property>
   <property name="text">
    <string>FFT</string>
   </property>
   <property name="toolTip">
    <string>Show the FFT of the image</string>
   </property>
   <property name="statusTip">
    <string>Show the FFT of the image</string>
   </property>
  </action>
  <action name="actionAcquire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="camera-web"/>
   </property>
   <property name="text">
    <string>Acquire</string>
   </property>
   <property name="toolTip">
    <string>Acquire an image from the camera</string>
   </property>
   <property name="statusTip">
    <string>Acquire an image from the camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SacHistoWidget</class>
   <extends>QWidget</extends>
   <header>sacvision.histowidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sacvision.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
